.github/workflows/build.yml

```
@@ -5,6 +5,8 @@ on:
     types:
       - opened
       - synchronize
+    paths-ignore:
+      - docs/**
```

for the new changes:

```suggestion
      - docs/**
      - logo/**
      - *.md
```